Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj

```diff
@@ -42,37 +34,27 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\</OutputPath>
     <DefineConstants>TRACE;DEBUG;HAVE_OPENTK</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <AndroidLinkMode>Full</AndroidLinkMode>
-    <JavaMaximumHeapSize>1G</JavaMaximumHeapSize>
-    <EmbedAssembliesIntoApk>True</EmbedAssembliesIntoApk>
+    <AndroidLinkMode>None</AndroidLinkMode>
+    <EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>
+    <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
```

Comment:
Is this condition not working? https://github.com/xamarin/Xamarin.Forms/blob/7d77bb5043abb765701296bfd20d5087a014a7ed/Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj#L61
I don’t understand why you need to do this unless the CI env var isn’t set. @rmarinho?

Reply:
The CI property is set inside Directory.build.props which isn't pulled in until after this target runs

https://github.com/xamarin/Xamarin.Forms/blob/7d77bb5043abb765701296bfd20d5087a014a7ed/Xamarin.Forms.ControlGallery.Android/Xamarin.Forms.ControlGallery.Android.csproj#L380

So it's currently *CI* is just blank where it's at at line 61

This PR moves the CI property group lower in the csproj so that it takes effect

https://github.com/xamarin/Xamarin.Forms/pull/7834/files#diff-f0d93698c5f1681c6d6aa0451b8f7782R381

Reply:
I thought it was set in yaml, but I never checked. If it was there it would be set at MSBuild evaluation time.

Reply:
Or just use TF_Build

https://github.com/xamarin/Xamarin.Forms/blob/master/Directory.Build.props#L24

Not sure if there's a reason for using our own *$(CI)* property over just using $(TF_Build) inside the non project SDK csproj files

Reply:
@rmarinho any reason we can't just use *$(TF_Build)* ?

Reply:
Nop, is just because it could change I the future, other CI for example, so using CI was a way to be agnostic of the CI used